modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/utils/JsonPathSpec.scala

```diff
@@ -88,4 +93,23 @@ class JsonPathSpec extends Specification {
 
   def e6 =
     JsonPath.query("$.store.book[2]", Json.fromString("somestring")) must beRight(List())
+
+  def e7 =
+    JsonPath.anyToJson(null) must be(Json.Null)
+
+  def e8 = {
+    JsonPath.anyToJson(5) must beEqualTo(Json.fromLong(5))
```

Comment:
This one won't have an effect - you can change it to `JsonPath.anyToJson(5) must beEqualTo(Json.fromLong(10))`

Reply:
Implicit widening.